shared/config.lua

```diff
@@ -27,6 +27,9 @@ end
 -- Enables Dynamic Doors
 Config.DynamicDoors = false
 
+-- If you set it to true, anyone you allow access to your property will also be able to access your garage.
+Config.AllowAccessToGarage = false
```

Comment:
Potential var rename to make it more clear its for all ppl who have access?
```suggestion
Config.AllowGarageAccessToAllResidents = false
```

Reply:
or AllowGarageAccessForResidents
or AllowResidentsGarageAccess